### PR TITLE
Change syntax highlight theme

### PR DESCRIPTION
**Pull request by marksdk**

### CODE REVIEW

File: src/_sass/syntax-theme.scss [hunk 1]
```diff
@@ -1,128 +1,190 @@
...
- }
-
- .token.entity {
-     cursor: help;
- }
\ No newline at end of file
+/* xml */
+/* java */
+/* kotlin */
+/* swift */
```
Comment (simonlaustsen) [inline src/_sass/syntax-theme.scss:187]: Does this mean that syntax-specific colors are missing for the following?

**Review by simonlaustsen — APPROVED** [review]
I relay started liking the dark themed code blocks 😢  

Added a single question otherwise approved 👍

File: src/_sass/syntax-theme.scss (hunk 1 above)
Comment (marksdk) [reply src/_sass/syntax-theme.scss:187]: Ah, sorry. It means that we could add them if we want, as these are the languages we use throughout the site, but don't currently have specific styles for. I don't know enough about different conventions of highlights to know if we should support a different style for each of these languages, so I just put them in. We can keep them or remove them—I'm fine with either 🙂

Comment (simonlaustsen) [reply src/_sass/syntax-theme.scss:187]: Let's keep them for now 👍

**Pull request merged**